match2/commons/get_match_group_copy_paste.lua

```diff
@@ -10,7 +10,12 @@ local Lua = require("Module:Lua")
 local BracketAlias = Lua.moduleExists("Module:BracketAlias") and mw.loadData('Module:BracketAlias') or {}
 local WikiSpecific = require("Module:GetMatchGroupCopyPaste/wiki")
 
-local args
+function copyPaste.generateID()
+	--initiate the rnd generator
+	math.randomseed(os.time())
+	math.random(); math.random(); math.random()
```

Comment:
I presume we're throwing away the values since `os.time()` is in seconds? Can we just use `os.clock()` (CPU time) and not waste CPU time?

Reply:
the throwing the 3 rnd numbers is actually just a habit which increases the randomness
as we actually do not need it to be real rnd here, but only to get the id, we can completely kick the throwing away